component-library/src/app/pages/banner-documentation/banner-documentation.component.html

```diff
@@ -0,0 +1,743 @@
+<div class="content-doc-wrapper">
+  <app-title-slug-url [config]="pageTitleSlugConfig"></app-title-slug-url>
+  <p class="body1 intro">{{ 'Banner.Intro' | translate }}</p>
+
+  <section class="interactive-demo">
+    <app-title-slug-url
+      [config]="headingConfig.interactiveDemoSlugConfig"
+    ></app-title-slug-url>
+    <p class="body3 demo">{{ 'Banner.InteractiveDemoDesc' | translate }}</p>
```

Comment:
Instead of passing `class="body3` or `class="h6"` we can use `@include typography.fontSet(body, 3, regular);` or `@include typography.fontSet(heading, 6, regular);`

Reply:
Fixed